airbyte-cdk/java/airbyte-cdk/s3-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/s3/util/Stringify.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.integrations.destination.s3.util
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import kotlin.jvm.Throws
+
+enum class Stringify(val value: String) {
```

Comment:
NIT: Why an enum rather than a boolean? I feel like it would simplify a bunch of conditionals, and I don't foresee adding another value to the enum (which would be enough to justify putting an enum here)